lib/tasks/publishing_api.rake

```diff
@@ -188,6 +188,26 @@ namespace :publishing_api do
     puts "Finished queuing items for Publishing API"
   end
 
+  desc "Republish all editions with official attachments"
+  task republish_editions_with_official_attachments: :environment do
+    editions = Edition.where(
+      id: (
+        (
+          Attachment.where.not(unnumbered_hoc_paper: [nil, false]).pluck(:attachable_id) + Attachment.where.not(unnumbered_command_paper: [nil, false]).pluck(:attachable_id) + Attachment.where.not(command_paper_number: nil).pluck(:attachable_id) + Attachment.where.not(hoc_paper_number: nil).pluck(:attachable_id)
+        ).uniq
+      ),
+    )
+    puts "# Sending #{editions.count} editions to Publishing API"
+
+    editions.pluck(:id).each_with_index do |item_id, i|
+      PublishingApiLinksWorker.perform_async(item_id)
```

Comment:
This worker doesn't republish the edition, it just patches the links for the relevant document. Is that sufficient for what you want to change?

Reply:
I hadn't realised this, I've changed it so it now invokes the document republishing worker.